### ISSUE TITLE
ignore some html tags like tables, p, br, etc.

### ISSUE THREAD

**Issue by emmapeel2** [numericube/lektor-i18n-plugin#13]
the strings generated include lots of html tags that could be stripped. for example the sentences:
```

#: http://localhost/transports/ (content/transports/contents+en.lr:topic.body)
msgid "<table>"
msgstr "<table>"
 
#: http://localhost/transports/ (content/transports/contents+en.lr:topic.body)
msgid "<tbody>"
msgstr "<tbody>"

```
dont need to be translated.

**Comment by johnzhou721** [comment]
Hmm... maybe translate by paragraph and don't put blank lines after or before it? And also if you're using Weblate or any translation tool that allows glossaries, mark those tags as untranslatable so people won't accidetnally trasnlate it.

**Comment by emmapeel2** [comment]
Yes, I can do that. Or maybe you can also do it and so all the users of the plugin will benefit as well?
